### DISCUSSION TITLE
Can I use the web UI to run slurm commands?

### DISCUSSION THREAD

**Discussion by frosk1** [rackslab/Slurm-web#365] [Q&A] [answered]
Hey,

I have a question about the web UI.
Is it possible to run slurm commands and start jobs with the ui?

**Comment by rezib** [comment, marked as answer]
Hello @frosk1, I just converted your issue #364 into a discussion as it is just a question, so it is better placed here.

You cannot run any Slurm command such as `sinfo` or `squeue` directly in the web UI. The web UI is intended to present the same level of information of these commands without requiring users to know these commands.

For the moment, Slurm-web v3 does not give the possibility to submit jobs but it is planned in the roadmap with #281.
